rmgpy/molecule/kekulize.pyx

```diff
@@ -52,12 +52,13 @@ and the process continues until the entire molecule can be solved.
 
 import logging
 
+from .graph cimport Graph, Vertex, Edge
 from rmgpy.exceptions import KekulizationError, AtomTypeError
 from rmgpy.molecule.element import PeriodicSystem
 from rmgpy.molecule.molecule cimport Atom, Bond, Molecule
```

Comment:
Molecule is now an unused import in this file.

Reply:
Done!